.travis.yml

```diff
@@ -31,4 +31,6 @@ install:
 
 script:
     - pip install virtualenv==1.10
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install "mercurial>=3.3" ; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install python-hglib==1.5 ; fi
```

Comment:
> I wasn't sure how to indicate that Mercurial would need to come from system packages if it's a Python 3 test.

I think `apt-get install mercurial` should work.  But if `python-hglib` doesn't work on Python 3 yet, the mercurial support here wouldn't work anyway, right?  So perfectly fine to leave as-is for now until the Python 3 support is ready.